packages/flame/lib/src/assets/images.dart

```diff
@@ -108,7 +123,12 @@ class Images {
     return loadAll(imagePaths.toList());
   }
 
-  /// Convert an array of pixel values into an [Image] object.
+  /// Waits until all currently pending image loading operations complete.
+  Future<void> ready() async {
+    await Future.wait(_assets.values.map((asset) => asset.retrieve()));
+  }
```

Comment:
`imagesLoaded` was the name suggestion I got from the Flutter code review channel.

Reply:
But then the usage code will be
```dart
await images.imagesLoaded;
```
which sounds tautological. And if we remove `images`, then `images.loaded` sounds as if it could be a boolean indicator.

Reply:
That is really weird now when I think about it, `Images.imagesLoaded` makes no sense at all.

Reply:
We must have commented on pretty much the same time. :smile: 

I think `loaded` and `ready` sounds as much as boolean indicators both of them, but the person in the code review channel also mentioned that according to Effective Dart that kind of boolean would have an `is` prepended to it:
https://dart.dev/guides/language/effective-dart/design#prefer-a-non-imperative-verb-phrase-for-a-boolean-property-or-variable
I thought that I read somewhere that they wanted you to remove the `is` and `has` from names for Dart, but apparently that was only for parameter names.